### PR TITLE
Removed legacy UI classes and layouts.

### PR DESCRIPTION
**Pull request by jamesjmtaylor**
This PR removes all the legacy activities, fragments, and xml layouts.  I'm not too worried about removing things we might want to reference later because we can always checkout older commits.  My larger concern is developers new to the project making changes to the incorrect ui module. Let me know if I missed anything.

### CODE REVIEW

File: app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt [hunk 1]
```diff
@@ -83,7 +83,7 @@ class BluetoothManagerImpl(
     private fun foregroundNotification(): Notification {
         createNotificationChannelIfNeeded()
 
-        val notificationIntent = Intent(app, MainActivity::class.java)
+        val notificationIntent = Intent(app, TempActivity::class.java)
```
Comment (Apisov) [inline app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt]: `TempActivity` was a temporary measure while we had `MainActivity`. It's better if you just rename `TempActivity` to `MainActivity` and the code that depended on the old `MainActivity` won't simply change. 

ie we wouldn't need to rename all references to `MainActivity` to `TempActivity`

Comment (Apisov) [reply app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt]: Clean all code related to this, including `AndroidManifest.xml`

**Review by Apisov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt (hunk 1 above)
Comment (jamesjmtaylor) [reply app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt]: Done.

**Review by nkumarcc — APPROVED** [review]
This lgtm. I wanted to give `develop` a test drive before we merged because of the crashes people have been seeing. Looks like we should be set!

**Comment by nkumarcc** [comment]
Btw doing this because it looks like you've hit @Apisov's request :)

**Pull request merged**